compiler/resolution/functionResolution.cpp

```diff
@@ -3514,24 +3534,29 @@ void resolveNormalCall(CallExpr* call) {
   if (call->partialTag && (!best || !best->fn ||
                            !best->fn->hasFlag(FLAG_NO_PARENS))) {
     if (best != NULL) {
-      delete best;
+      // best is deleted below with the other candidates
       best = NULL;
```

Comment:
I added a print in this section and did a full test run. It appears to be dead code. I'll add a comment to that effect.